<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTExampleClass</class>
 <widget class="QWidget" name="QTExampleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTExample</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>281</width>
     <height>391</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLineEdit" name="idlineEdit"/>
   <widget class="QLineEdit" name="namelineEdit"/>
   <widget class="QLineEdit" name="grouplineEdit"/>
   <widget class="QLineEdit" name="gradelineEdit"/>
   <widget class="QLineEdit" name="teacherlineEdit"/>
   <widget class="QPushButton" name="addPushButton">
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removepushButton">
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitPushButton">
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTExample.qrc"/>
 </resources>
 <connections/>
</ui>
